client/src/components/App.jsx

```diff
@@ -23,18 +24,32 @@ class App extends React.Component {
         this.setState({ reviews: reviews.data });
       })
       .catch((error) => {
-        // eslint-disable-next-line no-console
         console.log(error);
       });
   }
 
+  handleSortedReviews(sort) {
+    console.log(sort)
+    axios.get('/80/reviews', {
```

Comment:
Why 80? Is there a way to not have the restaurant_id hardcoded in? Maybe have it passed into the handleSortedReviews function as an argument?